src/main/java/com/abdullah/coding/challenge/util/TimeUtil.java

```diff
@@ -0,0 +1,32 @@
+package com.abdullah.coding.challenge.util;
+
+import java.sql.Timestamp;
+import java.text.SimpleDateFormat;
+
+public class TimeUtil {
+
+    private TimeUtil() {
+    }
+
+    public static boolean isValidTimeStamp(String time) {
+        return time.length() == 16;
```

Comment:
1. What's the validation logic here?

Reply:
Simply, I expect from a client to send me a date in a particular format and that format must have 16 characters. Not much of an importance.